test/layering_test.dart

```diff
@@ -0,0 +1,48 @@
+//  Copyright 2023 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+import 'package:layerlens/src/analyzer.dart';
+import 'package:layerlens/src/model.dart';
+import 'package:test/test.dart';
+
+import 'test_infra/tests.dart';
+
+final _tests = <String, Dependencies>{
```

Comment:
This Map could live inside the test_infra/tests.dart file too

Reply:
This map is for this test only while test_infra contains cross-test data.
Added comment to test_infra.